plugins/admin/core/root.go

```diff
@@ -25,8 +25,9 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"knative.dev/client-contrib/plugins/admin/pkg"
 	"knative.dev/client-contrib/plugins/admin/pkg/command"
+	autoscaling "knative.dev/client-contrib/plugins/admin/pkg/command/autoscaling"
 	"knative.dev/client-contrib/plugins/admin/pkg/command/domain"
-	private_registry "knative.dev/client-contrib/plugins/admin/pkg/command/registry"
+	registry "knative.dev/client-contrib/plugins/admin/pkg/command/registry"
```

Comment:
Why are you combining changes to `registry` with this PR which is titled `... enable or disable scale-to-zero`?

Unless these "registry" changes are needed for scale-to-zero I would move them to a different PR.

Reply:
@maximilien  This change is not required here. Will update it in another PR related to registry. Thanks!